src/com/goide/sdk/GoSdkService.java

```diff
@@ -74,7 +75,7 @@ public static String getGoExecutablePath(@Nullable String sdkHomePath) {
         // gcloud and standalone installations
         return sdkHomePath.endsWith(GoConstants.GCLOUD_APP_ENGINE_DIRECTORY_PATH)
                ? FileUtil.join(StringUtil.trimEnd(sdkHomePath, GoConstants.GCLOUD_APP_ENGINE_DIRECTORY_PATH), "bin", executable)
-               : FileUtil.join(sdkHomePath, executable);
+               : FileUtil.join(sdkHomePath, SystemInfo.isWindows ? StringUtil.replace(executable, ".exe", ".bat") : executable);
```

Comment:
Sorry for the delay, gcloud windows installer took like forever, and `gcloud components update gae-go`wasn't fast either.

OK, this is amazing, GAE for Windows uses `goapp.bat`but gcloud for Windows uses `goapp.cmd`.

For gcloud it should be `? FileUtil.join(StringUtil.trimEnd(sdkHomePath, GoConstants.GCLOUD_APP_ENGINE_DIRECTORY_PATH), "bin", SystemInfo.isWindows ? StringUtil.replace(executable, ".exe", ".cmd") : executable)`, so the return statement would be
```java
        return sdkHomePath.endsWith(GoConstants.GCLOUD_APP_ENGINE_DIRECTORY_PATH)
               ? FileUtil.join(StringUtil.trimEnd(sdkHomePath, GoConstants.GCLOUD_APP_ENGINE_DIRECTORY_PATH), "bin", SystemInfo.isWindows ? StringUtil.replace(executable, ".exe", ".cmd") : executable)
               : FileUtil.join(sdkHomePath, SystemInfo.isWindows ? StringUtil.replace(executable, ".exe", ".bat") : executable);
```

Should I create a new pull request which includes gae and gcloud for Windows correction?